src/component/admin/apiToken/ApiTokenForm/SelectProjectInput.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { Fragment, useMemo, useState, ChangeEvent } from 'react';
+import {
+    Link,
+    Checkbox,
+    FormControlLabel,
+    TextField,
+    Box,
+    Paper,
+} from '@material-ui/core';
+import {
+    Autocomplete,
+    AutocompleteRenderInputParams,
+    AutocompleteRenderOptionState,
+} from '@material-ui/lab';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import { IAutocompleteBoxOption } from 'component/common/AutocompleteBox/AutocompleteBox';
+import { useStyles } from './ApiTokenForm.styles';
+
+const ALL_PROJECTS = '*';
+
+// Fix for shadow under Autocomplete - match with Select input
+const CustomPaper = ({ ...props }) => <Paper elevation={8} {...props} />;
+
+export const SelectProjectInput = ({
+    options,
+    defaultValue = [ALL_PROJECTS],
+    onChange,
+    disabled,
+    error,
+    onFocus,
+}: {
```

Comment:
We usually declare props in a standalone interface, named something like `ISelectProjectInputProps`.